core/unit_test/TestViewOutOfBoundsAccess.hpp

```diff
@@ -125,6 +125,8 @@ void test_view_out_of_bounds_access() {
   TestViewOutOfBoundAccess(make_view<V6>(lbl), exec_space, prefix + ".*" + lbl);
   TestViewOutOfBoundAccess(make_view<V7>(lbl), exec_space, prefix + ".*" + lbl);
   TestViewOutOfBoundAccess(make_view<V8>(lbl), exec_space, prefix + ".*" + lbl);
+  // We check the memory space for unmanaged Views when configuring with Kokkos_ENABLE_DEBUG
+#ifndef KOKKOS_ENABLE_DEBUG
```

Comment:
What is not legal here?  The construction is fine, just as we do in `TestViewEmptyRuntimeUnmanaged.hpp`.
Is your point that we can construct these views but not used them?
Why would that out of bounds check fail in debug mode?

Reply:
We are constructing a non-empty unmanged View from a nullptr (that is detected as host pointer). We can just provide a valid allocation here instead, though.

Reply:
I pushed a better fix.